lib/fileOperations.js

```diff
@@ -196,11 +198,55 @@ function writeFile(path, data, options, callback) {
   }
 }
 
+function mkdirp(dirpath, mode, callback) {
+  if (typeof mode === 'function') {
+    callback = mode;
+    mode = undefined;
+  }
+
+  var m = mode || DEFAULT_DIR_MODE;
+  var cb = callback || function() {};
+  dirpath = path.resolve(dirpath);
+
+  fs.mkdir(dirpath, m, function(er) {
```

Comment:
>  This probably shouldn't be using mkdirp at all
This pull request removes the 'mkdirp' external dependency.  The only mkdirp is this function defined in fileOperation.js.